### DISCUSSION TITLE
date input for getOwnClassTimetableForRange()

### DISCUSSION THREAD

**Discussion by Falcon23K** [SchoolUtils/WebUntis#66] [Q&A] [answered]
I don't know how to put the date for this or any other function in. Can someone please help me?

**Comment by luwol03** [comment, marked as answer]
You can just pass normal js date objects to the functions. Under the hood is a function which [converts js date objects into untis dates](https://webuntis.noim.me/index.js.html#line260). This function is also exported from the libary. [See](https://webuntis.noim.me/WebUntis.html#.convertUntisDate)

**Reply by TheNoim** [reply]
Exactly.
